httpclient5-cache/src/main/java/org/apache/hc/client5/http/impl/cache/DefaultAsyncCacheRevalidator.java

```diff
@@ -130,6 +130,12 @@ public AsyncDataConsumer handleResponse(
                                     return asyncExecCallback.handleResponse(response, entityDetails);
                                 }
 
+                                @Override
+                                public void handleInformationResponse(final HttpResponse response) throws HttpException, IOException {
+                                    responseRef.set(response);
```

Comment:
@kirill-usov Are you sure this is the right thing to do here? Should informational response have any impact on cache re-validation?